pyvista/plotting/renderer.py

```diff
@@ -466,12 +467,12 @@ def disable_anti_aliasing(self):
         self.SetUseFXAA(False)
         self.Modified()
 
-    def add_border(self, color=(1, 1, 1), width=2.0):
+    def add_border(self, color='w', width=2.0):
```

Comment:
'white' is explicit than 'w' :)
```suggestion
    def add_border(self, color='white', width=2.0):
```